app/app.py

```diff
@@ -1,167 +1,113 @@
+import jwt
+import csv
+import logging
 import os
+import yaml
+import torch
 import pandas as pd
 from dotenv import load_dotenv
-from keras.preprocessing.sequence import pad_sequences
-from utils.logger import get_logger 
+from fastapi import FastAPI, HTTPException, Depends, status
 from pydantic import BaseModel
-from fastapi import FastAPI, Depends, HTTPException, status
-from fastapi.security import (
-    OAuth2PasswordBearer,
-    OAuth2PasswordRequestForm,
-)
-from passlib.context import CryptContext
-from typing import Union
-from datetime import datetime, timedelta, timezone
-import jwt
-from jwt import PyJWTError
-from tensorflow.keras.models import load_model
-import joblib
+from transformers import BertTokenizer, BertForSequenceClassification
+from src.api.security import authenticate_user
```

Comment:
Hello Prince, Here in line 12 in `app.py` file you are import from module `src.api.security`, but in the current pushed code the module doesn't exist. If we pushed this code to the development branch it will cause some errors in the main code.

Reply:
Yes, great you pointed that out. This is due to my local directory structure. I'll work on it.

Reply:
I think I made that changes in my current commit. Can you confirm that?